client/src/SignIn.js

```diff
@@ -93,11 +93,19 @@ function SignIn({ config, smtpConfigured, passport, refreshAppContext }) {
     </div>
   );
 
+  const samlForm = (
+    <div>
+      <a href={config.baseUrl + '/auth/saml'}>Sign in with ADFS</a>
+    </div>
+  );
+
   return (
     <div style={{ width: '300px', textAlign: 'center', margin: '100px auto' }}>
       <h1>SQLPad</h1>
+      {JSON.stringify(passport.strategies)}
```

Comment:
I think this was maybe accidentally left from debugging?

Reply:
Ahh, yes, overlooked it during cleanup.